### PR TITLE
Blackjack

### PR DESCRIPTION
**Pull request by f8daniel**
Fixes the two main issues in #207, namely (1) processes special payment rules for blackjacks and has dealer auto-win on a blackjack and (2) imposes limits on bets so that players can't purposefully* bet more money than they have and lose the game when they go bankrupt. There is now an endgame state you can play down to with a clearly announced winner.

*Because blackjacks result in double payments and one player acts as the house staking all the other players wager, one can still go into debt under certain circumstances.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,16 +12,19 @@ logs/*
...
 config/modules.config.js
+config/runtime.config.js
+
+
 
 .DS_Store
 node_modules
 node_modules/*
 tests/config/env
-package-lock.json
```
Comment (escapedcat) [inline .gitignore:17]: ❤️

Comment (f8daniel) [reply .gitignore:17]: So I obviously didn't change these config files, crazy how that might have happened

Comment (escapedcat) [reply .gitignore:17]: Hm, you didn't? I like those. Wondering were all these changes are coming form then. Anyways ;)

**Comment by arpee** [comment]
Not sure either - this may have been a side effect of me... doing stuff.

r

Richard Parris - Co-Founder

ph : +86 1860 1221 974

**Pull request merged**